### DISCUSSION TITLE
Cookie notice

### DISCUSSION THREAD

**Discussion by keebrev** [mkdocs/mkdocs#2346] [Q&A]
Since there's a feature for implementing Google Analytics quite easy, I was wondering if there's also an easy option for enabling cookie notices? Adding the GA tag would require a cookie notice and approval of the end-user.

**Comment by waylan** [comment]
> Adding the GA tag would require a cookie notice and approval of the end-user.

I'm assuming this "requirement" is a legal requirement in some jurisdictions? The current implementation predates any such requirements and you are the first to bring this up. A PR which adds the notice is certainly welcome. Although, it would probably be best if analytics was broken out into a plugin. Then there could easily be analytics plugins for various services rather than only builtin support for one such service.

**Comment by fu-sen** [comment]
I agree with the opinion of plug-in. It's a feature that MkDocs doesn't necessarily need. I added to Wiki's Plugins Ideas:
https://github.com/mkdocs/mkdocs/wiki/Plugins-Ideas
